codespell_lib/data/dictionary.txt

```diff
@@ -1983,6 +1986,7 @@ basf->base
 basicaly->basically
 basicly->basically
 batery->battery
+batter->better
```

Comment:
Add battery.

Reply:
Done

Reply:
It's not there @Gelma . Lost in your conflict fixing?

Reply:
Done. Sorry, I forgot because I fixed other points. Now it's done.